pkg/apis/operators/v2/catalogsourceconfig_types.go

```diff
@@ -79,6 +79,12 @@ func (csc *CatalogSourceConfig) EnsureGVK() {
 	csc.SetGroupVersionKind(gvk)
 }
 
+// ForceReconcile forces the CatalogSourceConfig object to be reconciled.
+func (csc *CatalogSourceConfig) ForceReconcile() {
```

Comment:
Thinking a bit for about this, I suggest `ForceUpdate()`.

Reply:
Sure.